### DISCUSSION TITLE
Do Google publish details of how they will further rollout 3rd party cookie restriction beyond the 1%

### DISCUSSION THREAD

**Discussion by Dory998** [privacysandbox/privacy-sandbox-dev-support#370] [General]
Concern I have is that pocket of my customer/user population will find their browsers change one day and features stop working before we've tested them. Is it a case of a gradual ramp up from 1 to 100%?
As of January 4, 2024, Google has started restricting third-party cookies by default for 1% of Chrome browsers. This change is part of their plan to facilitate testing and gradually ramp up to 100% of users by the third quarter of 2024.

**Comment by samdutton** [comment]
Hi @Dory998 — thanks for your question.

The timeline for third-party cookie phaseout in Chrome [has been updated](https://privacysandbox.com/news/update-on-the-plan-for-phase-out-of-third-party-cookies-on-chrome/).

[privacysandbox.com/open-web](https://privacysandbox.com/open-web/) shows the current timeline.

The [1% testing period](https://developers.google.com/privacy-sandbox/blog/cookie-countdown-2024jan) continues through the second half of 2024. Subject to resolving any competition concerns, Chrome plans to begin rolling out third-party cookie phaseout to all Chrome clients starting from early 2025.

Users in the 1% group can temporarily allow third-party cookies for a site by clicking the eye icon in Chrome's address bar.

<img width="993" alt="image" src="https://github.com/privacysandbox/privacy-sandbox-dev-support/assets/205226/d860782b-df5d-4bb3-933b-a87040912589">
<img width="993" alt="image" src="https://github.com/privacysandbox/privacy-sandbox-dev-support/assets/205226/f08a0c60-145e-4fe3-9378-16f56cdef00a">

**Reply by Dory998** [reply]
@samdutton thanks for the info sam. am I right in thinking the 1% will remain as this and no further populations will be added to? thanks

**Comment by samdutton** [comment]
Hi @Dory998 

>  am I right in thinking the 1% will remain as this and no further populations will be added to?

Subject to resolving any competition concerns, Chrome plans to begin rolling out third-party cookie phaseout from early 2025, eventually ramping up to 100% of Chrome clients.

![image](https://github.com/privacysandbox/privacy-sandbox-dev-support/assets/205226/59119f21-516b-49ef-adb2-91ab21ddfeed)
